static/app/views/organizationCreate/index.tsx

```diff
@@ -59,7 +86,18 @@ function OrganizationCreate() {
             stacked
             required
           />
-
+          {should_display_regions() && (
+            <SelectField
+              name="region"
+              label="Data Storage"
+              help="Where will this organization reside?"
+              choices={get_region_choices()}
+              onChange={setRegion}
+              stacked
+              inline={false}
+              required
```

Comment:
I can't get the form to respect this being required for the life of me, so I'm open to suggestions on this one 😅 

Worst case, would it be possible to default this to the first choice?

Reply:
> Worst case, would it be possible to default this to the first choice?

Defaulting this dropdown to US makes sense to me.

Reply:
That seems fine to me as well, at least in the short term, we can always revisit as a part of EA/GA